firebase-firestore/ktx/ktx.gradle

```diff
@@ -56,7 +56,7 @@ dependencies {
     testImplementation 'com.fasterxml.jackson.core:jackson-databind:2.9.8'
     testImplementation 'com.google.android.gms:play-services-tasks:16.0.1'
     testImplementation 'com.google.protobuf:protobuf-lite:3.0.1'
-    testImplementation "org.robolectric:robolectric:$robolectricVersion"
```

Comment:
Would it make sense to upgrade robolectricVersion in the root build.gradle file for it to apply to all projects?